doc/howto/network_bridge_firewalld.md

```diff
@@ -93,12 +91,28 @@ For example:
     sudo ufw route allow in on lxdbr0
     sudo ufw route allow out on lxdbr0
 
+````{warning}
 % Repeat warning from above
 ```{include} network_bridge_firewalld.md
     :start-after: <!-- Include start warning -->
     :end-before: <!-- Include end warning -->
 ```
 
+Here's an example for more restrictive firewall rules that limit access from the guests to the host to only DHCP and DNS and allow all outbound connections from IPv4 hosts:
+
+```
+# allow the guest to get an IP from the LXD host
+sudo ufw allow in on lxdbr0 to any port 67 proto udp
```

Comment:
I couldn't easily find if `ufw` allowed specifying an IPv4 only rule but we should add a corresponding one for IPv6 (UDP/547).